frontend/src/Editor/Editor.jsx

```diff
@@ -1493,6 +1493,8 @@ class Editor extends React.Component {
                             runQuery={this.runQuery}
                             dataSourceModalHandler={this.dataSourceModalHandler}
                             setStateOfUnsavedQueries={this.setStateOfUnsavedQueries}
+                            appDefinition={appDefinition}
```

Comment:
Can you check and remove 'appDefinition' prop if it is not used in `QueryManager.jsx` file?

Reply:
Hey appdefinition ( this for querymanager) is used in modalactions in `apputils.js` @kavinvenkatachalam

Reply:
Hey appdefinition ( this for querymanager) is used in modalactions in `apputils.js` @kavinvenkatachalam